tests/test_app_ctrl.py

```diff
@@ -0,0 +1,113 @@
+from lake.models.app_ctrl_model import AppCtrlModel
+from lake.modules.app_ctrl import AppCtrl
+from lake.passes.passes import lift_config_reg
+import _kratos
+import magma as m
+from magma import *
+import fault
+import tempfile
+import kratos as kts
+import random as rand
+import pytest
+
+
+def test_app_ctrl(int_in_ports=1,
+                  int_out_ports=3):
+
+    # Set up model..
+    model_ac = AppCtrlModel(int_in_ports=int_in_ports,
+                            int_out_ports=int_out_ports)
+
+    new_config = {}
+    new_config['input_port_0'] = 0
+    new_config['input_port_1'] = 0
+    new_config['input_port_2'] = 0
+    new_config['read_depth_0'] = 196
+    new_config['read_depth_1'] = 196
+    new_config['read_depth_2'] = 196
+    new_config['write_depth_0'] = 196
+
+    model_ac.set_config(new_config=new_config)
+    ###
+
+    # Set up dut...
+    dut = AppCtrl(interconnect_input_ports=int_in_ports,
+                  interconnect_output_ports=int_out_ports)
+
+    lift_config_reg(dut.internal_generator)
+    magma_dut = kts.util.to_magma(dut, flatten_array=True,
+                                  check_multiple_driver=False,
+                                  check_flip_flop_always_ff=False)
```

Comment:
delete line 37 and add the pass in to_magma like additional_passes={"lift config regs": lift_config_reg}?

Reply:
Also just noticed a fair amount of the top level tests do that, so I think we can change it everywhere. Some tests seem fine already.